_docs/privacy/web-tracking-protections.md

```diff
@@ -286,6 +280,34 @@ We review user-reported Broken Site Reports daily, and work to address these usa
+
+For example, a domain that hosts open-source JavaScript libraries would be considered a CDN and thus not blocked by 3rd-Party Tracker Loading Protection, but any instance of the FingerprintJS fingerprinting library hosted on that domain would be blocked because it’s used to fingerprint you and track you around the web. Likewise, 3rd-Party Tracker Loading Protection doesn’t apply to Google Tag Manager, as we have not detected the tag manager itself participating in tracking, we’ve observed it associated with breakage on many sites, and it occasionally loads necessary features like site widgets and privacy-friendly analytics libraries. However, we continue to block tracking resources Google Tag Manager may load, such as Google Analytics, and we also block Google Tag Manager itself on sites where we’ve detected embedded fingerprinting code within it. Other Infrastructure Domains are treated similarly.
+
+| Platform      | Support                                                                                                         |
+| ------------- | --------------------------------------------------------------------------------------------------------------- |
+| All platforms | Infrastructure Domain categorization impacts how we build our tracker lists, which are shared across platforms. |
+
+### DuckDuckGo Private Search Ads
+
+Advertising on DuckDuckGo is done in partnership with Microsoft. Viewing ads on DuckDuckGo is anonymous, and Microsoft has [committed][ads-by-microsoft] to not profile our users on ad clicks: “when you click on a Microsoft-provided ad that appears on DuckDuckGo, Microsoft Advertising does not associate your ad-click behavior with a user profile. It also does not store or share that information other than for accounting purposes.”
+
+Previously, we were limited in how we could apply our 3rd-Party Tracker Loading Protection on Microsoft tracking scripts due to a policy requirement related to our use of Bing as a source for our private search results. This is no longer the case and as of October 2022 we block Microsoft scripts from loading on 3rd-party websites in our browsing apps (MacOS, Windows, iOS, and Android) and in our browser extensions (Chrome, Firefox, Safari, Edge and Opera). To be clear, Microsoft scripts were never embedded in our search engine or apps, which do not track you. Companies insert these scripts on their own websites for their own purposes, and so they never sent any information to DuckDuckGo.
```

Comment:
```suggestion
Previously, we were limited in how we could apply our 3rd-Party Tracker Loading Protection on Microsoft tracking scripts due to a policy requirement related to our use of Bing as a source for our private search results. This is no longer the case and as of October 2022 we block Microsoft scripts from loading on 3rd-party websites in our browsing apps (MacOS, Windows, iOS, and Android) and in our browser extensions (Chrome, Firefox, Safari, Edge, and Opera). To be clear, Microsoft scripts were never embedded in our search engine or apps, which do not track you. Companies insert these scripts on their own websites for their own purposes, and so they never sent any information to DuckDuckGo.
```

Reply:
I know this was pre-existing and not you, but as long as we're here can we throw this one missed comma in?? 😬

Reply:
+1 to oxford commas

Reply:
+1